app/assets/javascript/app/assets/styles/components/ContributorForm.scss

```diff
@@ -0,0 +1,182 @@
+.component.ContributorForm {
+  @extend .abstract.Form;
+  position: fixed;
+  z-index: 5;
+
+  height: 100%;
+  width: 100%;
+
+  &:after {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    display: block;
+
+    height: 100%;
+    width: 100%;
+
+    background: $color_bg-medium;
+    opacity: .5;
+  }
+
+  .form-box {
+    @extend ._vertical-center;
+    @extend ._clear;
+    top: 30%;
+    max-width: 600px;
+    margin: $global_pad auto;
+    border: 1px solid $color_brand-secondary;
+    background: $color_bg-light;
+    z-index: 10;
+    .form-header {
+      background: $color_brand-secondary;
+      padding: ($global_pad / 2) $global_pad;
+      h2 {
+        margin: 0;
+        color: $color_bg-light;
+      }
+      a {
+        @extend ._vertical-center;
+        left: inherit;
+        right: $global_pad / 2;
+
+        margin: auto;
+
+        color: $color_brand-secondary--dark;
+        font-size: 14px;
+      }
+    }
+    p.form-description {
+      margin: 0;
+      padding: $global_pad;
+    }
+    .contributor-list-wrapper {
+      max-height: 300px;
+      border-width: 1px 0 1px 0;
+      border-style: solid;
+      border-color: $color_bg-medium;
+      overflow: scroll;
+    }
+    ul.contributor-list {
+      margin: 0;
+      padding: ($global_pad / 2) 0;
+    }
+    li.contributor {
+      padding: ($global_pad / 2) $global_pad;
+      .contributor-details {
+        display: flex;
+        &.placeholder p {
+          margin: 0;
+          padding: ($global_pad / 2) 0;
+        }
+        input.contributor-link {
+          flex: 1;
+        }
+        & > *:not(:last-child) {
+          margin-right: $global_pad / 2;
+        }
+      }
+      p.contributor-warning {
+        color: $state_negative;
+        margin: 8px 0 0 0;
+      }
+    }
+    .contributor-add {
+      display: flex;
+      padding: $global_pad;
+      label {
+        // margin: ($global_pad / 2) 0;
+        font-size: 14px;
+        font-weight: 300;
+        line-height: 28px;
+      }
+      input {
+        flex: 1;
+      }
+      & > *:not(:last-child) {
+        margin-right: $global_pad / 2;
+      }
+    }
+  }
+
+
+
+
+
+
+  // &:after {
```

Comment:
Is this commented out CSS being saved for something or can we delete it?

Reply:
Yup. Don't need that. Thanks!